cogs/send_introduction_reminders.py

```diff
@@ -232,12 +234,13 @@ async def opt_out_introduction_reminders_button_callback(self, button: discord.B
             )
 
             _BUTTON_WILL_MAKE_OPT_IN: Final[bool] = bool(
-                    button.style == discord.ButtonStyle.green
-                    or str(button.emoji) == emoji.emojize(
-                        ":raised_hand:",
-                        language="alias",
-                    )
-                    or button.label and "Opt back in" in button.label)
+                button.style == discord.ButtonStyle.green
+                or str(button.emoji) == emoji.emojize(
+                    ":raised_hand:",
+                    language="alias",
+                )
+                or button.label and "Opt back in" in button.label,
```

Comment:
Remove trailing comma and replace with `# noqa: COM812`

Reply:
Which one? both?

Reply:
Just this one on line 42: `or button.label and "Opt back in" in button.label  # noqa: COM812`